src/components/App/App.tsx

```diff
@@ -21,11 +26,47 @@ class App extends React.Component<Props> {
     .then(data => this.setState({apiList: data.entries})) 
   }
 
+  filter = (stateObj: FilterState):any => {
+    let matchingCards: Api[] = [];
+    if (this.state) {
```

Comment:
should this be if(this.state.search)??

Reply:
Turns out we don't need the conditional at all